src/components/Transaction/Form.tsx

```diff
@@ -27,6 +27,7 @@ import {
   WalletProviderOpts
 } from '../../services'
 import { useLogger } from '../../services/EnvironmentProvider'
+import { WarnPotentialDup } from './PotentialDup'
```

Comment:
Just to be a nitpicker I'd prefer the component and filename to match, either are fine for me